server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -262,9 +262,9 @@ export function create(
 	router.post(
 		"/:tenantId/document/:id/revokeToken",
 		validateRequestParams("tenantId", "id"),
+		throttle(perTenantThrottler, winston, tenantThrottleOptions),
```

Comment:
Why not tenantThrottler() middleware?

Also, let's also add perCluster throttle for this api.

Reply:
Changed to perTenantGeneralThrottler, and this throttler is not enabled. :)

Do we need to throttle this API? Maybe create a work item to enable the throttle for this API?